examples/experimental/onnx/mobilenet_v2/main.py

```diff
@@ -63,7 +64,7 @@ def run_example():
     input_name = onnx_model.graph.input[0].name
     def transform_fn(data_item):
         images, _ = data_item
-        return {input_name: images.numpy()}
+        return {input_name: ONNXNNCFTensor(images.numpy())}
```

Comment:
This change is not a user friendly. Could you revert it?

Reply:
Reverted.